Makefile

```diff
@@ -0,0 +1,20 @@
+TEMURIN_TAGS := 8u322-b06-jdk 11.0.14.1_1-jdk 8u322-b06-jre 11.0.14.1_1-jre
+JOBS := $(addprefix temurin-,${TEMURIN_TAGS})
+CHECKS := $(addprefix check-,${TEMURIN_TAGS})
+CLEAN := $(addprefix clean-,${TEMURIN_TAGS})
+
+all: ${JOBS}
+check: ${CHECKS}
+.PHONY: all ${JOBS}
+
+${JOBS}: temurin-%: Dockerfile
+	docker build \
+	  --build-arg TEMURIN_TAG=$* \
+	  --tag dwolla/docker-java:$*-SNAPSHOT \
+	  .
+
+${CHECKS}: check-%:
+	grep --silent "^          - $*$$" .github/workflows/ci.yml
+
+${CLEAN_JOBS}: clean-%:
+    docker rmi dwolla/docker-java:$*-SNAPSHOT
```

Comment:
It looks like this is indented with spaces instead of a tab, so make doesn't recognize this as a command under the `${CLEAN_JOBS}` targets.

I think we may also want to add the `--force` option to the command as well, so that it succeeds even if the image doesn't exist. (That would let the user run `make clean all` and have it always build the images from scratch—without the flag, the `clean` target fails, and the images don't get built.)